fedimint-client/src/lib.rs

```diff
@@ -131,50 +228,93 @@ impl Debug for ClientInner {
     }
 }
 
+/// Global state given to a specific client module and state. It is aware inside
+/// which module instance and operation it is used and to avoid module being
+/// aware of their instance id etc.
+#[derive(Clone, Debug)]
+struct ModuleGlobalClientContext {
+    client: Arc<ClientInner>,
+    module_instance: ModuleInstanceId,
```

Comment:
nit: I'd prefer `module_instance_id` here, I got confused what this was when looking at the code

Reply:
Will try to stay consistent in the future/fix in another PR.